Makefile

```diff
@@ -51,19 +51,19 @@ setup: ## Setup local environment for development, generate configuration files.
 
 build: ## Builds all containers or the one specified (n=service name)
 	@echo "$(P) Building images..."
-	@docker-compose -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml build --no-cache $(n)
+	@docker-compose -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile core --profile kafka build --no-cache $(n)
```

Comment:
We only want the core and Kafka services to startup by default.  The others are not critical for development, and some of the utilities will fail startup if they are not done manually after everything else is running.